api/synapse-api-rest-imperative/src/main/java/io/americanexpress/synapse/api/rest/imperative/controller/BaseDeleteImperativeRestController.java

```diff
@@ -33,9 +32,9 @@ public class BaseDeleteImperativeRestController<
     @DeleteMapping
     @ResponseStatus(HttpStatus.NO_CONTENT)
     @Operation(tags = "Delete Operation", summary = "Deletes a resource")
-    public void delete(@RequestHeader HttpHeaders headers, @Valid @RequestBody I serviceRequest) {
```

Comment:
For my understanding, why is the @Valid not needed anymore? Does the serviceRequest not have any validation annotations?

Reply:
it was moved to service, since its the model for the service not the controller.

https://github.com/americanexpress/synapse/blob/feature/java21-spring6-boot3-cleanup/service/synapse-service-imperative/src/main/java/io/americanexpress/synapse/service/imperative/service/BaseService.java#L41